virttest/iscsi.py

```diff
@@ -843,7 +843,10 @@ def create_iSCSI(params, root_dir=data_dir.get_tmp_dir()):
             raise exceptions.TestError("Failed to install iscsi initiator"
                                        " packages")
         # Install linux iscsi target software targetcli
-        iscsi_package = ["targetcli"]
+        if distro.detect().version == '18' and distro.detect().name == 'Ubuntu':
```

Comment:
This would break in future Ubuntu releases,
```python
if distro.detect().name == 'Ubuntu' and int(distro.detect().version) >= 18:
    iscsi_package = ["targetcli-fb"]
else:
    iscsi_package = ["targetcli"]
```